src/main/kotlin/com/sourcegraph/cody/agent/protocol_extensions/TestingParamsUtil.kt

```diff
@@ -0,0 +1,6 @@
+package com.sourcegraph.cody.agent.protocol_extensions
+
+object TestingParamsUtil {
```

Comment:
Could also be `TestingParamsExt` for consistency

Reply:
👍